DEVELOPER_GUIDE.md

```diff
@@ -16,10 +16,23 @@ We depend upon the version of node defined in [.nvmrc](.nvmrc).
 
 You will probably want to install a node version manager. [nvm](https://github.com/creationix/nvm) is recommended.
 
-To install and use the correct node version with `nvm`:
+To install and use the correct node version v with `nvm`:
 
+v - verion mentioned in [.nvmrc](.nvmrc).
+
+```
+nvm install v
```

Comment:
You actually don't need to specify a version if you're in the same directory as the `.nvmrc` file. `nvm install` is sufficient enough